public/pcpg_cancercell_2017/meta_study.txt

```diff
@@ -0,0 +1,6 @@
+type_of_cancer: soft_tissue
+cancer_study_identifier: pcpg_cancercell_2017
+name: Pheochromocytoma and Paraganglioma (Cell, 2017)
```

Comment:
let's follow the convention here too: `Pheochromocytoma and Paraganglioma (TCGA, Cancer Cell 2017)`